test/resources/test-flavor spaces/real-test-flavor/real_flavor_base/flavor_test_build_run/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM {{base_test_build_run}}
+
+RUN mkdir /conf /buckets
+
+COPY --from={{flavor_customization}} /usr /usr
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{flavor_customization}} /opt /opt
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{flavor_customization}} /lib /lib
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{flavor_customization}} /bin /bin
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{flavor_customization}} /etc /etc
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{flavor_customization}} /build_info /build_info
+RUN true # workaround for https://github.com/moby/moby/issues/37965
```

Comment:
is this a jinja template? if so one could generate this with a loop, passing the path(s) as parameter from within or from the outside of the template, e.g.:
```jinja
{% for path in ['/usr', '/opt', '/lib'] %}
COPY --from={{flavor_customization}} {{ path}} {{ path }}
RUN true # workaround for https://github.com/moby/moby/issues/37965
{% endfor %}
```